workflows/kube-proxy-all/workflow.yaml

```diff
@@ -110,6 +110,8 @@ spec:
                               value: '500'
                             - name: TOTAL_CHAOS_DURATION
                               value: '60' # in seconds
+                            - name: CHAOS_KILL_COMMAND
+                              value: 'kill $(find /proc -name exe -lname '*/dd' 2>&1 | grep -v 'Permission denied' | awk -F/ '{print $(NF-1)}' |  head -n 1)'
```

Comment:
Please use double quotes @ispeakc0de it is giving error while parsing with single quotes.